### PR TITLE
Deployment workflow

### LINKED ISSUE #1: deploy using ansible to nest2.bocoup.com
Following the instructions @ https://github.com/bocoup/infrastructure/wiki/Creating-Servers

### PR DESCRIPTION
**Pull request by jennschiffer**
closes #1 

* sets up vagrant/ansible to deploy locally at nest2.loc
* server is set up at nest2.bocoup.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,3 +54,101 @@ required features may not be available in older versions.
 * `test` - lint + unit test (auto-runs via travis-ci)
 * `unit` - run all unit tests with mocha
 * `unit:watch` - run `unit` and watch for changes
+
+
+## Manual Deployment
+
+### Test-deploying a local commit (Vagrant)
+
+1. Run `./deploy/run-playbook.sh deploy vagrant local=true commit=_____` where
```
Comment (cowboy) [inline README.md]: You may want to mention that once this command has been run, the Vagrant box will no longer be serving up the local `public` folder, but will be serving a build of the specified commit from within the Vagrant box. To revert Vagrant back to serving up the local `public` folder, this command will need to be run:

```
$ ./deploy/run-playbook.sh vagrant-link vagrant
```

(`vagrant-link` runs as part of `vagrant provision` via the `init` playbook, which is why that's the default behavior)

File: README.md [hunk 2]
```diff
@@ -6,6 +6,19 @@ The following will need to be installed on your local development machine before
 you can use this workflow. All versions should be the latest available, as some
 required features may not be available in older versions.
 
+* **[Ansible](http://docs.ansible.com/) 1.9.2**
```
Comment (tkellen) [inline README.md]: Can we bump this to Ansible 2.x ?

Comment (cowboy) [inline README.md]: Can the README make it more clear at what point in the development process which dependencies are needed? I don't want people to think they need to install all these things to just modify some styles.

Eg.

* Node is required for development
* + Ansible is required for deploying to staging or production
* + VirtualBox, Vagrant and the vagrant-hostsupdater plugin are required for testing the build process

File: README.md [hunk 3]
```diff
@@ -6,10 +6,31 @@ The following will need to be installed on your local development machine before
 you can use this workflow. All versions should be the latest available, as some
 required features may not be available in older versions.
 
+### Required for Development
+
 * **[node.js 5.5.x](https://nodejs.org/)**
   - [Download](https://nodejs.org/) (All platforms)
   - Install `node` via [homebrew][homebrew]
 
+### Required for Deployment to Staging or Production
```
Comment (cowboy) [inline README.md]: What if this and the next header said "Also Required..."?

**Pull request closed without merging**